docs/utils/mdHelper.js

```diff
@@ -28,13 +29,56 @@ export async function getDocByRoute(route: string): Promise<{|
   }
 }
 
-export async function getAllMarkdownPosts(): Promise<Array<string>> {
+export async function getAllMarkdownPosts(): Promise<Array<string[]>> {
+  function convertNamesForURL(name: string): string {
+    return name.replace(/ /g, '_').replace(/'/g, '').toLowerCase();
+  }
+
   const docsDirectory = path.join(process.cwd(), 'markdown');
 
-  const folderContents = await fs.readdir(docsDirectory);
-  const cleanedNames = folderContents
-    .filter((name) => name.endsWith('.mdx') || name.endsWith('.md'))
-    .map((name) => name.replace('.md', ''));
+  const getAllSitePaths = () => {
+    const pagePaths = [];
+
+    const addUrlPaths = (pageItems: Array<siteIndexType | string>, _pages: Array<string>) => {
+      const pages: Array<string> = _pages.concat([]);
```

Comment:
What is `.concat([])` doing here? That should be effectively a no-op?

Reply:
Good catch. That was me doing trying to solve some pass by reference issues. We don't need it anymore. Removed.

Reply:
fixed in #2227